make/RunTests.gmk

```diff
@@ -60,8 +60,13 @@ define SetTestOpt
   endif
 endef
 
+AGENT_PREFIX := lib
+AGENT_EXT := .so
```

Comment:
We already have `LIBRARY_PREFIX` and `SHARED_LIBRARY_SUFFIX` defined in spec.gmk. To make those available in a test prebuilt scenario, they need to also be defined in RunTestPrebuiltSpec.gmk. Then you can use those instead of defining these.